cypher-manual/clauses/union.md

```diff
@@ -59,7 +59,7 @@ Output:
 
 ## 2. Combine queries and remove duplicates
 
-To combine two or more queries and return their results without removing duplicates, use the `UNION` clause without `ALL`.
+To combine two or more queries and return their results while removing duplicates, use the `UNION` clause without `ALL`.
```

Comment:
The smaller grammar/typo fixes should be also copied back to all of the released version in which they exist. Am I right @g-despot ?

Reply:
Is this cypher-manual documentation dependent on specific release? A quick search give me a single hit for the phrase "To combine two or more queries and return their results without removing duplicates, use the `UNION` clause without `ALL`.".
(I have looked in /docs)